client/src/java/com/zimbra/client/ZMailbox.java

```diff
@@ -5744,6 +5746,15 @@ public com.zimbra.soap.mail.type.CalendarItemInfo getRemoteCalItemByUID(String r
         return resp == null ? null : resp.getItem();
     }
 
+    public com.zimbra.soap.mail.type.CalendarItemInfo getRemoteCalItemByID(String requestedAccountId, String id,
```

Comment:
why are we using a fully qualified name here?

Reply:
Yes not required. Removed and used as import